llama_stack/providers/inline/inference/meta_reference/inference.py

```diff
@@ -76,6 +81,12 @@ def check_model(self, request) -> None:
     async def unregister_model(self, model_id: str) -> None:
         pass
 
+    async def register_model(self, model: Model) -> Model:
+        model = await self.model_registry_helper.register_model(model)
+        if model.model_type == ModelType.embedding_model:
+            self._get_embedding_model(model.provider_resource_id)
```

Comment:
this method call says the method should be named `_load_sentence_transformer_model()`

Reply:
done.